packages/sources/dxfeed/src/endpoint/price.ts

```diff
@@ -14,7 +14,18 @@ export const batchablePropertyPath = [{ name: 'base', limit: 120 }]
 const customError = (data: any) => data.Response === 'Error'
 
 export const inputParameters: InputParameters = {
-  base: ['base', 'from', 'coin', 'market'],
+  base: {
+    required: true,
+    aliases: ['from', 'coin', 'market'],
+    description: 'The symbol of the currency to query',
+    type: 'string',
+  },
+  overrides: {
```

Comment:
This one is a base input parameter that is shared across every EA.

Probably don't need it here.

Reply:
Does this apply to these other properties as well?
```
const excludableInternalAdapterRequestProperties = [
  'resultPath',
  'overrides',
  'tokenOverrides',
  'includes',
]

Reply:
Yeah definitely. We should convert those to use the new input parameter syntax to give them descriptions.

Reply:
Ticket: https://app.shortcut.com/chainlinklabs/story/25085/add-references-to-shared-inputs-in-each-ea-readme
For now, I'll just remove these properties from every endpoint's input parameters.